src/java/main/org/apache/zookeeper/server/PrepRequestProcessor.java

```diff
@@ -628,14 +628,7 @@ protected void pRequest(Request request) throws RequestProcessorException {
                 break;
  
             //All the rest don't need to create a Txn - just verify session
-            case OpCode.sync:
-            case OpCode.exists:
-            case OpCode.getData:
-            case OpCode.getACL:
-            case OpCode.getChildren:
-            case OpCode.getChildren2:
-            case OpCode.ping:
-            case OpCode.setWatches:
+            default:
```

Comment:
This will execute if any `unknown type` and is not expected, isn't it?
We could keep the existing case checks and add default like,
```
                zks.sessionTracker.checkSession(request.sessionId,
                          request.getOwner());
                break;
            default:
                LOG.warn("unknown type " + request.type);
                break;

```